### PR TITLE
Momersion indicator and test added.

### PR DESCRIPTION
**Pull request by Jay-Jay-D**
Jared and Michael, here is the Momersion indicator implementation and the respective test.

### CODE REVIEW

File: Indicators/Momersion.cs [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                Mc = _multipliedDiffWindow.Count(obs => obs > 0);
+                MRc = _multipliedDiffWindow.Count(obs => obs < 0);
+                momersion = 100m * Mc / (Mc + MRc);
+            }
+            else
+            {
+                momersion = 50m;
+            }
+
+            Current = new IndicatorDataPoint(input.Time, momersion);
```
Comment (mchandschuh) [inline Indicators/Momersion.cs]: No need to set `Current` within the `ComputeNextValue` functions on indicators. They're designed such that you just return the new value and the base class will update the current

File: Tests/Indicators/MomersionTest.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
+﻿using NUnit.Framework;
```
Comment (mchandschuh) [inline Tests/Indicators/MomersionTest.cs]: Needs banner here

**Comment by mchandschuh** [comment]
@Jay-Jay-D - awesome stuff! Will pull in later tonight, thanks!

**Comment by Jay-Jay-D** [comment]
I'm so glad to help, thank you!

**Comment by bizcad** [comment]
Hi JJ,

 

I hope you have a good trip.  Here is an interesting idea.

 

http://archive.aweber.com/awlist3824363/4iANX/h/How_to_trade_the_equity_curve.htm

 

Nick

**Comment by mchandschuh** [comment]
Merged in https://github.com/QuantConnect/Lean/commit/9c2e8b03868d1724a9f1303a04a592b892465fc6

**Pull request closed without merging**